meerk40t/gui/wxmribbon.py

```diff
@@ -470,6 +470,11 @@ def sort_priority(elem):
             b.identifier = button.get("identifier")
             b.action = button.get("action")
             b.action_right = button.get("right")
+            if "rule_enabled" in button:
+                b.enable_rule = button.get("rule_enabled")
+            else:
+                b.enable_rule = lambda cond: True
+
```

Comment:
Dictionaries `.get()` takes `key, default` so you can just do `b.enable_rule = button.get("rule_enabled", lambda cond: True)`

Though the preference for PEP8 etc is to avoid lambda if you can (for actual functions, this is just a quicky fake function and clearly great for that) but `None` is probably preferable and just check if `b.enable_rule` is not None. This is how it's working for "action" and "right".

Also if the rule_enable could vary between toggle/untoggle state or multi button (drop downs like camera) then it would need to save in the aspect and get a bit more complex and would need to check on state change.